gitsync/handler.py

```diff
@@ -54,15 +55,28 @@ async def send_response():
     async def process_push(self, ref_name: str, ref_head: bytes, force: bool) -> bool:
         # TODO: Virtual branch refs/for/XXX
         # TODO: Avoid conflict -> cancel pipeline
-        # Fetch local objects
+        
+        # Convert refs/head/* to refs/bmeta/*
```

Comment:
Users should push to "bmeta" directly. It's your responsibility to change `bmeta` corresponding to `head` whenever a user pushes.

Reply:
So we reject when users push to `refs/head/*`?

Reply:
Should *not*. I'm sorry I made a typo ...